packages/ui/src/components/Requirement/Requirement.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactText } from 'react';
+import { Flex } from '@chakra-ui/react';
+
+import { SolidCheckIcon } from '../../icons/SolidCheck';
+import { Typography } from '../Typography';
+
+export type RequirementProps = {
```

Comment:
💭 I'm not 100% sure, but probably you can extend this type with `FlexProps`, restructure props to the Flex element and remove `Box` elements you used in `MentorRequirements.tsx` to wrap those requirements (pass all props to the Requirement element).